src/ecephys_etl/modules/align_timestamps/probe_synchronizer.py

```diff
@@ -122,32 +126,37 @@ def compute(
             sample index of probe acquisition start time
         local_probe_sampling_rate : float
             the probe's apparent sampling rate
-    
+
         Returns
         -------
-        ProbeSynchronizer : 
-            When called, applies the transform computed here to samples on the probe clock.
+        ProbeSynchronizer :
+            When called, applies the transform computed here to samples on
+            the probe clock.
 
         """
 
         times_array = np.array(probe_barcode_times)
         barcodes_array = np.array(probe_barcodes)
 
-        ok_barcodes = np.where((times_array > min_time) * (times_array < max_time))[0]
+        ok_barcodes = np.where(
+            (times_array > min_time) * (times_array < max_time)
```

Comment:
Can we make this an `np.logical_and`. Multiplying booleans this way, while legal, is confusing to me.